syncthing-bep/src/main/kotlin/net/syncthing/java/bep/BlockPuller.kt

```diff
@@ -84,22 +88,27 @@ object BlockPuller {
                             // is handled differently so that the timeout is ignored.
                             // Due to that, it's converted to an IOException.
 
-                            throw IOException("timeout during requesting block")
+                            val exception = IOException("Time limit exceeded while requesting block.")
+                            LOGGER.atInfo().withThrowable(exception).log("Timeout set for {} seconds.", box(timeoutInMillis/1000))
```

Comment:
careful with things like `timeoutInMillis/1000`. You're going to lose accuracy, because the math is done entirely in whole numbers. You need to explicitly ask for decimals if you want them. In this case, just log it as millis. Using seconds isn't a big benefit.

spaces around operators like `/`

Again, if we're throwing an exception, it's an `error` in the logs